lib/collection.js

```diff
@@ -373,3 +374,16 @@ Collection.prototype.update = function (query, update, opts, fn) {
     })
   })({update: update, query: query, options: opts, callback: fn}, 'update')
 }
+
+Collection.prototype.replaceOne = function (query, replacement, opts, fn) {
```

Comment:
could it be an option on `findOneAndUpdate` instead of expanding the API surface?

Reply:
Thankyou for your comment. I have added the changes. Please check it.